modular_skyrat/modules/cyborg/code/mechafabricator_designs.dm

```diff
@@ -52,3 +52,15 @@
 	)
 	category = list(RND_CATEGORY_CYBORG_UPGRADE_MODULES)
 
+/datum/design/sign_upgrade
+	name = "Cyborg Upgrade (Nano-Metallic Picket Sign)"
+	id = "sign_module"
+	construction_time = 60
```

Comment:
This should be using a time define not a raw number value

Reply:
all construction_time instances use a raw number value.